src/config/dynamic-theme-fixes.config

```
@@ -4958,6 +4958,15 @@ header a[href="/"] svg {
 
 ================================
 
+doc.qt.io
+
+CSS
+body {
+  background-color: var(--darkreader-neutral-background) !important;
+}
```

```suggestion
:root {
    --content-bg-color: var(--darkreader-neutral-background) !important;
}
```

I think it's better to fix the css variable that's used.

---

Thanks!